start

```diff
@@ -439,6 +457,20 @@ function exec_supervisor() {
     2> >(sed 's/^/supervisor: /' >&2)
 }
 
+function print_service_logs() {
+  # Wait for supervisord to be up.
+  while ! supervisorctl pid > /dev/null ; do
+    sleep 1;
+  done
+  # Start tailing logs from notable services.
+  supervisorctl tail -f stellar-core stdout > >(sed "s/^/${purple}stellar-core | $clear/") &
```

Comment:
slick! not suggesting a change, just noticed supervisor's  `stdout_syslog=true` and `stderr_syslog=true` settings, in that case, wouldn't need to do the while loop wait on supervisor possibly missing some early logs emitted by services in interim, each service log pipe changes slightly to like:
```
 tail -f /var/log/syslog | grep stellar-core | > >(sed "s/^/${purple}stellar-core | $clear/") &
```

but haven't tried this, is just idea

Reply:
We don't have syslog installed or setup in the image currently. We could set it up, although that probably requires some investigation.